pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -491,23 +511,6 @@ class StateVectorKokkos final
         applyMultiQubitOp(matrix_, wires, inverse);
     }
 
-    /**
-     * @brief Apply a given matrix directly to the statevector.
-     *
-     * @param matrix Matrix data (in row-major format).
-     * @param wires Wires to apply gate to.
-     * @param inverse Indicate whether inverse should be taken.
```

Comment:
Any reason to remove this method?

Reply:
There is a similar method below, just with `const std::vector<ComplexT> &matrix` instead of `std::vector<ComplexT> &matrix`, so the one without const here shouldn't be necessary?